frontend/admin/src/js/components/apiManagedTable/ApiManagedTable.stories.tsx

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import { Meta, Story } from "@storybook/react";
+
+import { FromArray } from "@common/types/utilityTypes";
+
+import { action } from "@storybook/addon-actions";
+import { fakeUsers } from "@common/fakeData";
+import BasicTable from "./BasicTable";
+import TableHeader from "./TableHeader";
+import TableFooter from "./TableFooter";
+import {
+  ColumnsOf,
+  handleColumnHiddenChange,
+  IdType,
+  SortingRule,
+} from "./basicTableHelpers";
+import { User } from "../../api/generated";
+
+type ApiManageTableProps = {
+  header: Omit<
+    React.ComponentProps<typeof TableHeader>,
+    | "onSearchChange"
+    | "onColumnHiddenChange"
+    | "hiddenColumnIds"
+    | "filterButtonComponent"
+    | "columns"
+  >;
+};
+
+const mockUsers = fakeUsers(100);
+
+type Users = Array<User>;
+
+type SearchState = {
+  term?: string;
+  type?: string;
+};
+
+type Data = NonNullable<FromArray<Users>>;
+
+const columns: ColumnsOf<Data> = [
+  {
+    label: "First Name",
+    id: "name",
+    accessor: (row: User) => row.firstName,
+    sortColumnName: "title",
```

Comment:
Should be firstName? (Also with lastName?)  It makes both columns show the sort icon together.

Reply:
Oops, copy and paste error :P

Fixed with [fix column id and sort name](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4542/commits/1268cc881f1e38842eb42b6b3ee7e689047b5c2d)